test/unit/connection/test-connection-ssl-min-tls1.0-accept.js

```diff
@@ -0,0 +1,31 @@
+var assert     = require('assert');
+var common     = require('../../common');
+var connection = common.createConnection({
+  port : common.fakeServerPort,
+  ssl  : {
+    ca         : common.getSSLConfig().ca,
+    minVersion : 'TLSv1'
+  }
+});
+
+var server = common.createFakeServer({
+  ssl: {
+    maxVersion: 'TLSv1'
+  }
+});
+
+server.listen(common.fakeServerPort, function(err) {
+  if (err) throw err;
+
+  connection.ping(function(err) {
+    assert.ifError(err);
```

Comment:
Ideally the test should assert that it is failing for the _reason_ we expect it to. Found quite a few bugs a while back when I added those kinds of tests in this lib. This assert is fine, but I'll typically add some additional asserts on the next couple lines to assert some relevant values like `err.code`, or `err.message` (whatever props are relevant here). I assume that `err.fatal` should be `true` here, so want to assert that.

Reply:
Since this is happy path test do we need to check the error message? Could fail for any number of reason, for happy path the test doesnt need to know why right?